### DISCUSSION TITLE
Which license is correct?

### DISCUSSION THREAD

**Discussion by StdEnku** [microsoft/WindowsAppSDK#3695] [Q&A]
The library is licensed under the MIT license on the repository side, while the Nuget package is licensed under the MICROSOFT SOFTWARE LICENSE TERMS.
Which license is correct?
Or do I have to comply with the obligations of both licenses when redistributing?

Thanks

**Comment by Yewzir** [comment]
@StdEnku 

Microsoft Software License Terms, by far.
